### ISSUE TITLE
Help with MatAutocompleteList

### ISSUE THREAD

**Issue by LuisAraya07** [SamProf/MatBlazor#412]
![Sin título](https://user-images.githubusercontent.com/39540911/73711900-f623c200-46cd-11ea-9454-399e8d16d6ff.png)

I need help with this I need Scroll here

**Comment by americanslon** [comment]
The whole point of auto complete as a concept is to not have a scroll IMHO. If you need a scroll why not just use a proper dropdown.

**Comment by enkodellc** [comment]
@americanslon I agree with you that it should have a height and if it goes past it then a vertical scroll.
